Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1203,12 +1203,8 @@ public string GetSmartApiUrl(string hostname, int? port = null)
         /// <inheritdoc/>
         public string GetLoopbackHttpApiUrl()
         {
-            if (NetManager.IsIP6Enabled)
-            {
-                return GetLocalApiUrl("::1", Uri.UriSchemeHttp, HttpPort);
-            }
-
-            return GetLocalApiUrl("127.0.0.1", Uri.UriSchemeHttp, HttpPort);
+            // Passing an external address cause GetBindInterface to return an externally accessible interface (if possible).
```

Comment:
Why external? It should be whatever local IP it is binding to.

Reply:
It's for multi-interface systems.  It doesn't have any effect on a single adapter system. (Hence the testing unit)

You mentioned it might be accessed via the client so the default is to use the external facing interface unless the user has specified otherwise.

Reply:
The "8.8.8.8" causes GetBindInterface to 'see' the client coming from an external address, so uses the interface that is best  to route to that location.